### ISSUE TITLE
[bitnami/sealed-secrets] chart missing necessary rbac roles

### ISSUE THREAD

**Issue by mateuszdrab** [bitnami/charts#13003]
### Name and Version

sealed-secrets-1.1.7

### What steps will reproduce the bug?

I documented this issue in https://github.com/bitnami-labs/sealed-secrets/issues/1017#issuecomment-1282124164

- deploy latest helm chart
- add a sealedsecret
- the error will be logged

### What is the expected behavior?

The secret should be unsealed - which happend only after manually adding the list permission to the clusterole

### What do you see instead?

```
W1015 18:17:35.756308       1 reflector.go:424] pkg/mod/k8s.io/client-go@v0.25.0/tools/cache/reflector.go:169: failed to list *v1.Secret: secrets is forbidden: User "system:serviceaccount:kube-system:sealed-secrets-controller" cannot list resource "secrets" in API group "" at the cluster scope
E1015 18:17:35.756413       1 reflector.go:140] pkg/mod/k8s.io/client-go@v0.25.0/tools/cache/reflector.go:169: Failed to watch *v1.Secret: failed to list *v1.Secret: secrets is forbidden: User "system:serviceaccount:kube-system:sealed-secrets-controller" cannot list resource "secrets" in API group "" at the cluster scope
```

### Additional information

It seems that the clusterrole is missing the list/watch permission for secrets - this prevents the sealed secrets controller from working.
I think this is related to the new feature implemented in https://github.com/bitnami-labs/sealed-secrets/pull/963

**Comment by fmulero** [comment]
I think it was fixed in #12970

**Comment by mateuszdrab** [comment]
That's correct. Thanks for the reminder.
